scripts/release_scripts/deploy.py

```diff
@@ -585,6 +610,8 @@ def execute_deployment():
             'install_third_party_libs.py prior to running this script.'
             % THIRD_PARTY_DIR)
 
+    verify_number_of_pending_deletion_requests(app_name)
```

Comment:
Wanted to confirm that we want to verify this before the deployment, right?

Reply:
Yes